CppCoreGuidelines.md

```diff
@@ -13393,8 +13393,29 @@ Application concepts are easier to reason about.
 
 ##### Example
 
-    ???
-
+    void publish(std::string* msg)
+    {
+        // ...
+        *msg = "Hello";
+        // ...
+    }
+    
+    void manual_publishing(std::string* msg)
+    {
+	    // Incapsulates thread functionality into the application task
```

Comment:
This line should use spaces instead of tabs. Also, I think you meant encapsulates instead of incapsulates.